gridsome.server.js

```diff
@@ -22,23 +22,25 @@ function TailwindPlugin(api, options) {
   const purgecss = require('@fullhuman/postcss-purgecss')(purgeConfig)
 
   api.chainWebpack(config => {
-    config.module
-      .rule('css')
-      .oneOf('normal')
-      .use('postcss-loader')
-      .tap(options => {
-        shouldImport && options.plugins.push(postcssImport)
+    ['css', 'scss', 'sass', 'less', 'stylus'].forEach(lang => {
+      config.module
+        .rule(lang)
```

Comment:
Why not use a regex instead? `/\.(css|p(ost)?css|scss|sass|less|styl(us)?)$/` would match [all of Gridsome supported style extensions](https://github.com/gridsome/gridsome/blob/master/gridsome/lib/webpack/createBaseConfig.js#L150-L155). It might also be a good idea to allow users to configure this regex in case they need to customize the settings for some of the languages.

The regex I suggested could be minimized but with this one we get to see we're supporting exactly the same extensions core does.

Reply:
I'm not sure where to use that regex, I've gone through the **webpack-chain** docs and as far as I can tell these rules are simple string that are already predefined and associated with a regex `test()` by Gridsome itself...adding a `test()` with a regex creates new rules instead of modifying existing ones I believe.